src/target/target.cc

```diff
@@ -1042,6 +1049,9 @@ TVM_REGISTER_GLOBAL("target.TargetExitScope").set_body_typed(TargetInternal::Exi
 TVM_REGISTER_GLOBAL("target.TargetCurrent").set_body_typed(Target::Current);
 TVM_REGISTER_GLOBAL("target.TargetExport").set_body_typed(TargetInternal::Export);
 TVM_REGISTER_GLOBAL("target.WithHost").set_body_typed(TargetInternal::WithHost);
+TVM_REGISTER_GLOBAL("target.TargetGetDeviceType").set_body_typed([](const Target& target) {
```

Comment:
do we need this or can we just use the node attr accessor?

Reply:
This function will check the attribute first, and if it's not set, it will return the default.  It also shortens the invocation of `GetAttr`.  Let me know what you'd prefer.